test/extended/util/client.go

```diff
@@ -331,14 +347,167 @@ func (c *CLI) SetupProject() string {
 		o.Expect(err).NotTo(o.HaveOccurred())
 	}
 
-	WaitForProjectSCCAnnotations(c.ProjectClient().ProjectV1(), newNamespace)
+	WaitForNamespaceSCCAnnotations(c.KubeClient().CoreV1(), newNamespace)
 
 	framework.Logf("Project %q has been fully provisioned.", newNamespace)
 	return newNamespace
 }
 
+func (c *CLI) setupNamespace() string {
+	requiresTestStart()
+	newNamespace := names.SimpleNameGenerator.GenerateName(fmt.Sprintf("e2e-test-%s-", c.kubeFramework.BaseName))
```

Comment:
why don't you use the kube framework method for namespace creation? It's likely to do most work for you.